### PR TITLE
Xena: Add playbooks & configuration for Cephadm

### PR DESCRIPTION
**Pull request by markgoddard**
Co-Authored-By: Michal Nasiadka <michal@stackhpc.com>

### CODE REVIEW

File: etc/kayobe/pulp.yml [hunk 1]
```diff
@@ -282,33 +282,64 @@ stackhpc_pulp_images:
...
-  {%- set _ = distributions.append(stackhpc_pulp_distribution_container_common | combine(distribution)) -%}
+  {%- set _ = distributions.append(stackhpc_pulp_distribution_container_kolla_common | combine(distribution)) -%}
   {%- endfor -%}
   {{ distributions }}
+
+# Whether to sync Ceph container images.
+stackhpc_sync_ceph_images: false
+
+# List of Ceph container image repositories.
+stackhpc_pulp_repository_container_repos_ceph:
```
Comment (mnasiadka) [inline etc/kayobe/pulp.yml:322]: Would it make sense to sync them through Ark for internet-constrained environments (just to add ark.stackhpc.com to whitelist and off you go).

**Review by mnasiadka — APPROVED** [review]

**Comment by mnasiadka** [comment]
Great work btw!

File: etc/kayobe/pulp.yml (hunk 1 above)
Comment (markgoddard) [reply etc/kayobe/pulp.yml:322]: Possibly, that means we'd need to setup syncing on Ark. Shall we put that on the backlog? :wink:

Comment (mnasiadka) [reply etc/kayobe/pulp.yml:322]: Yes please, mr Product Owner ;-)

**Pull request merged**